core/client/config/environment.js

```diff
@@ -33,7 +33,7 @@ module.exports = function (environment) {
         ENV.APP.LOG_TRANSITIONS_INTERNAL = true;
         ENV.APP.LOG_VIEW_LOOKUPS = true;
         ENV['ember-cli-mirage'] = {
-            enabled: false
+            enabled: true
```

Comment:
Is this line needed? ember-cli-mirage docs say it's enabled in development by default (unless the `--proxy` option is defined, which given our current setup it won't be).